app/models/user.rb

```diff
@@ -144,6 +144,19 @@ def as_json(options = {})
     super.tap { |h| h.key?('user') ? h['user']['name'] = name : h['name'] = name }
   end
 
+  apipie :class, desc: "A class representing #{model_name.human} object" do
+    sections only: %w[all additional]
+    property :id, Integer, desc: 'Numerical ID of the User'
+    property :firstname, String, desc: 'Returns the user first name'
+    property :lastname, String, desc: 'Returns the user last name'
+    property :description, String, desc: 'Returns the user description'
+    property :ssh_keys, array_of: SshKey, desc: 'Returns an array of the user\'s associated SshKey objects'
+    property :ssh_authorized_keys, array_of: SshKey, desc: 'Returns an array of strings representing the user\'s keys with their metadata (key, type and comment)'
```

Comment:
ssh_authorized_keys it's an array of strings containing the ssh key but not the SshKey object

Reply:
Yeap.. I wrote this in the description as well, but forgot to change the type... :facepalm: